t/author/01-confirm-schema.t

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env perl
+
+BEGIN {
+  unless ($ENV{AUTHOR_TESTING}) {
```

Comment:
You can also do something like this:
if (!$ENV{AUTHOR_TESTING}) {
    plan skip_all => 'Author test skipped';
}

Reply:
@waterkip resolved I think